ugc/ugc_tests/storage_tests.cpp

```diff
@@ -259,3 +264,42 @@ UNIT_TEST(StorageTest_LoadIndex)
   TEST(DeleteIndexFile(), ());
   TEST(DeleteUGCFile(), ());
 }
+
+UNIT_TEST(StorageTest_ContentTest)
+{
+  auto & builder = MwmBuilder::Builder();
+  m2::PointD const cafePoint(1.0, 1.0);
+  builder.Build({TestCafe(cafePoint)});
+  auto const cafeId = builder.FeatureIdForCafeAtPoint(cafePoint);
+  auto const oldUGC = MakeTestUGCUpdate(Time(chrono::hours(24 * 10)));
+  auto const newUGC = MakeTestUGCUpdate(Time(chrono::hours(24 * 300)));
+  Storage storage(builder.GetIndex());
+  storage.Load();
+  storage.SetUGCUpdate(cafeId, oldUGC);
+  storage.SetUGCUpdate(cafeId, newUGC);
+  TEST_EQUAL(storage.GetIndexesForTesting().size(), 2, ());
+  auto const toSendActual = storage.GetUGCToSend();
+
+  auto array = my::NewJSONArray();
+  string data;
+  {
+    using Sink = MemWriter<string>;
+    Sink sink(data);
+    SerializerJson<Sink> ser(sink);
+    ser(newUGC);
+  }
+
+  my::Json ugcNode(data);
+  auto embeddedNode = my::NewJSONObject();
+  ToJSONObject(*embeddedNode.get(), "data_version", cafeId.GetMwmVersion() );
+  ToJSONObject(*embeddedNode.get(), "mwm_name", cafeId.GetMwmName());
+  ToJSONObject(*embeddedNode.get(), "feature_id", cafeId.m_index);
+  ToJSONObject(*ugcNode.get(), "feature", *embeddedNode.release());
+  json_array_append_new(array.get(), ugcNode.get_deep_copy());
+  auto reviewsNode = my::NewJSONObject();
+  ToJSONObject(*reviewsNode.get(), "reviews", *array.release());
+  unique_ptr<char, JSONFreeDeleter> buffer(json_dumps(reviewsNode.get(), JSON_COMPACT | JSON_ENSURE_ASCII));
+  string const toSendExpected(buffer.get());
+  TEST_EQUAL(toSendActual, toSendExpected, ());
+  TEST(DeleteUGCFile(), ());
```

Comment:
All these tests call DeleteUGCFile. I suggest you to use UNIT_CLASS_TESTs, and delete ugc files in dtor.

Reply:
done